plugins/woocommerce-admin/client/marketplace/index.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './marketplace.scss';
+import Tabs from './components/tabs/tabs';
+import Content from './components/content/content';
+
+export default function Marketplace() {
+	const [ selectedTab, setSelectedTab ] = useState( 'discover' );
```

Comment:
I'm not sure if it matters, but when you manually change the URL to one that doesn't exist (such as the themes tab `wp-admin/admin.php?page=wc-admin&path=%2Fmarketplace&tab=themes`), I notice that Discover is the default but isn't highlighted. This is very nitpicky, and I have no preference here. Just mentioning in case it's something you're bothered about.

Reply:
Ah good spot, thanks! I've fixed this in [`e2b61c5` (#38885)](https://github.com/woocommerce/woocommerce/pull/38885/commits/e2b61c5b037653a983f999a57495ae2adcc921d2).